src/index.js

```diff
@@ -93,25 +93,15 @@ function injectToBaseOption(seriesNames, currentIndex, options) {
       },
       globe: {
         viewControl: {
-          autoRotate: false
+          autoRotate: true,
+          autoRotateAfterStill: 5,
+          distance: 300
         },
-        shading: "realistic",
+        shading: "color",
         baseTexture: WorldTopology,
         heightTexture: WorldTopology,
         displacementScale: 0.04,
-        environment: Starfield,
-        realisticMaterial: {
-          roughness: 0.9
-        },
-        light: {
-          ambient: {
-            intensity: 0.8,
-            shadow: true
-          },
-          main: {
-            intensity: 0.8
-          }
-        }
```

Comment:
Why do you think that removing all of the lighting effects is a good idea?

Reply:
Solve most compatibility issues and improve fluency on mobile devices